xradar/georeference/transforms.py

```diff
@@ -111,7 +112,18 @@ def get_x_y_z(ds, earth_radius=6371000, effective_radius_fraction=None):
 
     ds.z.attrs = {"standard_name": "height_above_ground", "units": "meters"}
 
-    # Make sure these
-    ds = ds.set_coords(["x", "y", "z"])
+    # Make sure the coordinates are set properly if it is a dataset
+    if isinstance(ds, xr.Dataset):
+        ds = ds.set_coords(["x", "y", "z"])
 
     return ds
+
+
+def get_x_y_z_tree(radar):
```

Comment:
Here, too, would be great to get the  re, ke keywords here too.

Reply:
Same as above. I can add that in, and we can get this merged!

Reply:
Yes, please merge at your discretion, @mgrover1!